app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.globant.example.mentorapp.view.activities;
+
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Window;
+
+import com.globant.example.mentorapp.R;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class SplashScreenActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Set portrait orientation
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        setContentView(R.layout.activity_splash);
+
+        TimerTask task = new TimerTask() {
+            @Override
+            public void run() {
+
+
+            }
+        };
+        Timer timer = new Timer();
+        timer.schedule(task, R.integer.splash_screen_duration);
```

Comment:
Keep in mind you're passing a resource id and not the value. You should use `getResources#getInteger()`

Double check this.